tasks/deploy/init.js

```diff
@@ -1,5 +1,6 @@
 var registerTask = require('../../lib/register-task');
 var getShipit = require('../../lib/get-shipit');
+var path = require('path2/posix');
```

Comment:
Ah - It was from this: https://github.com/timkelty/shipit-shared/pull/1, but I don't think we need it here.